### ISSUE TITLE
can't add tag

### ISSUE THREAD

**Issue by Belb1275** [Dijji/FileMeta#109]
hi,

I have windows 10 and have installed filemeta.  I have chosen .svg, .svgz and .png to add meta date too.  after restarting file explorer, i right click on the image and see line for tags, however it does not allow me to type anything in.  doesn't matter which file type i chose, nothing happens.  I have tried right clicking as well and removing and readding to this file type and nothing.

**Comment by Dijji** [comment]
Try selecting a .png file and right clicking it. Choose Properties. Select the Details tab in the dialog that comes up. You should see Tags in the Property column. Click to the right of it in the Value column. Does an editable field come up?

**Comment by Belb1275** [comment]
No it does not, that is my issue.  I got this program primarily for PNG and SVG files and it won’t let me work with them. I checked other files on my computer and it works just fine.

**Comment by Dijji** [comment]
Odd. I just tried it with .png files on my machine, and all works well. I extended the built-in Windows handler with the Simple profile. What existing handler are you seeing, and which profile did you choose? One last tip: sometimes a reboot is required to get everything working.

**Comment by Belb1275** [comment]
hi,

i used the simple profile as well.  i can see the tag option when right clicking on the png, but it wont let me enter anything.  i did restart explorer several times.  should i reboot the entire pc?

**Comment by Dijji** [comment]
Yes, rebooting does sometimes help.

In the Properties dialog, you need to hover with the mouse in the Value column to the right of the Tags entry in the first column. When you get to the right coordinates, a text box should appear.

Also, what was the original handler that File Meta extended for .png?

**Comment by Belb1275** [comment]
hI,

I get a text box for other things under details, but nothing appears
under tags for either png or svg now.  here is what my file meta screen
looks like.  Thank you for your help.

[image: image.png]

**Comment by sakkamade** [comment]
@Belb1275 Images are not upload via emails. Please do it in website.

**Comment by Belb1275** [comment]
![Uploading 98886F5E-DA28-4C91-83F6-870BA97598C0.png…]()

**Comment by Belb1275** [comment]
![4BA60F11-E9C2-432F-AFB9-1268C7A2B411](https://user-images.githubusercontent.com/77294640/105125189-c8d04f80-5aa9-11eb-9f5c-aeb3f197347b.png)

**Comment by Dijji** [comment]
That all looks fine. I'm starting to run out of ideas here. Have you rebooted since you configured File Meta as shown above?

**Comment by Belb1275** [comment]
Yes, I did.  And I removed and re-added the file type several times as well.

**Comment by resutora** [comment]
Check to see if you installed the x86 (32 bit) version by mistake, if you have a 64 bit machine. Had the exact same issue for all file types until I uninstalled and switched to x64. It's made clear on the wiki page that it won't function properly:

> This is the only way that the Explorer functionality will work because the Property Handler and Context Menu Handler run in the Explorer process, and so need to have the same bitness.
